pom.xml

```diff
@@ -229,6 +229,10 @@
 			<artifactId>scenery</artifactId>
 			<version>0.1.1-SNAPSHOT</version>
 		</dependency>
+		<dependency>
+			<groupId>net.imagej</groupId>
+			<artifactId>imagej-legacy</artifactId>
```

Comment:
If your goal is in fact to have imagej-legacy be inherited downstream, then I fear optional true will not due to trick for you. It explicitly requires downstream components to add the dependency themselves if they want it.